### PR TITLE
conformationpopups created

### PR DESCRIPTION
**Pull request by THIRUSELVAN-V**
completed conformationpopups

### CODE REVIEW

File: packages/ui/src/components/componentStyled/confirmationPopups/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	return (
+		<div className='bg-white w-[25rem] h-[15.875rem] rounded-borderRadius16px flex flex-col items-center justify-center'>
+			<div className=' flex flex-col items-center justify-around h-[14.375rem]'>
+				<div>{icon}</div>
+				<div className='flex flex-col items-center gap-2'>
+					<div className='font-medium text-base '>{heading}</div>
+					<div className='font-normal text-sm  text-center w-72 text-[#555555]'>
+						{subheading}
+					</div>
+				</div>
```
Comment (premCrayon) [inline packages/ui/src/components/componentStyled/confirmationPopups/index.tsx:31]: place footer button in this component, we don't need to pass as a pop

File: packages/ui/src/components/componentStyled/confirmationPopups/index.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	acceptButten?: React.ReactNode;
+	cancelButton?: React.ReactNode;
+}) => {
+	return (
+		<div className='bg-white w-[25rem] h-[15.875rem] rounded-borderRadius16px flex flex-col items-center justify-center'>
+			<div className=' flex flex-col items-center justify-around h-[14.375rem]'>
+				<div>{icon}</div>
+				<div className='flex flex-col items-center gap-2'>
+					<div className='font-medium text-base '>{heading}</div>
+					<div className='font-normal text-sm  text-center w-72 text-[#555555]'>
```
Comment (premCrayon) [inline packages/ui/src/components/componentStyled/confirmationPopups/index.tsx]: don't use  static color
use from theme
use font size also from theme

File: packages/ui/src/components/componentStyled/confirmationPopups/index.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+'use client';
+export const ConfirmationPopups = ({
+	icon,
+	heading,
+	subheading,
+	acceptButten,
+	cancelButton,
+}: {
+	icon?: React.ReactNode;
+	heading?: string;
```
Comment (premCrayon) [inline packages/ui/src/components/componentStyled/confirmationPopups/index.tsx]: declare typescript in interface

File: packages/ui/src/components/componentStyled/confirmationPopups/index.tsx (hunk 1 above)
Comment (THIRUSELVAN-V) [reply packages/ui/src/components/componentStyled/confirmationPopups/index.tsx:31]: completed

File: packages/ui/src/components/componentStyled/confirmationPopups/index.tsx (hunk 2 above)
Comment (THIRUSELVAN-V) [reply packages/ui/src/components/componentStyled/confirmationPopups/index.tsx]: completed

File: packages/ui/src/components/componentStyled/confirmationPopups/index.tsx (hunk 3 above)
Comment (THIRUSELVAN-V) [reply packages/ui/src/components/componentStyled/confirmationPopups/index.tsx]: completed

**Pull request merged**